x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-hipaa.adoc

```diff
@@ -225,13 +225,13 @@ Due to the differences between standard X12 and HIPAA schemas the use of overlay
 definition is not supported for HIPAA. Instead, for modifications, extract the supplied HIPAA schema from inside the `x12-schemas-2.0.0.jar`. This is found in the standard MuleSoft enterprise Maven repositories under group ID com.mulesoft.connectors.
 You can copy a message structure schema from this JAR (it contains both standard X12 and HIPAA schemas) and modify the extracted schema to use it directly. Unlike X12 schemas that use a base set of segment, composite, and element definitions, the HIPAA schemas are self-contained. This makes it easy to make changes to the schema without needing to work with multiple files.
 
-== About Recommended Types of validations from WEDI
+== About Recommended Types of Validations from WEDI
 
```

Comment:
GLOBALLY work to revise all the "About" titles....

Reply:
Introduce the two list items.

Reply:
Reformat these items to follow our bullet list style, as mentioned in earlier edit comments.